### DISCUSSION TITLE
Tagged PDF for accessibility

### DISCUSSION THREAD

**Discussion by nitinrajkumar451** [prawnpdf/prawn#1270] [Q&A]
I need to generate a pdf file with tags. How could I possibly achieve it?

**Comment by pointlessone** [comment]
Prawn doesn't support tagged PDF at the moment and there's no easy way to add it, unfortunately.

**Comment by gettalong** [comment]
Have a look at the following issue: https://github.com/prawnpdf/prawn/issues/1143 There you will find some more information and some ideas on how to implement it.
